README.md

```diff
@@ -86,57 +88,68 @@ Given this command, it will start a local development server and display the add
 
 You may supply a configuration file, in YAML format, to control various options.
 
-You can modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the validator. Also, you can provide the path to the configuration file name other than `.redocly.yaml` by using `--config` option when running the @redocly/openapi-cli.
+You can modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the linter. Also, you can provide the path to the configuration file name other than `.redocly.yaml` by using `--config` option when running the @redocly/openapi-cli.
 
-From a high-level, there are two configurable features: codeframes and rules.
-```yaml
-lint:
-  codeframes: on
-  rules:
-  ...
-```
+From a high-level, there are two configurable features: plugins and rules.
```

Comment:
what is a plugin?

Reply:
A plugin is a unified way to extend openapi-cli behavior (add custom rules, decorators, preprocessors, type extensions)

Reply:
Plugins can be local (local JS file) or npm package (we don't support it now but we will later).